pnpm-workspace.yaml

```diff
@@ -3,4 +3,5 @@ packages:
     - 'apps/**'
     - 'dapps/**'
     - '!**/dist/**'
+    - '!sdk/typescript/builder'
```

Comment:
for some reason manypkg will doesn't ignore this with just `'!sdk/typescript/*/**'`, but for turbo `!sdk/typescript/*` ignores the root typescript package.json